docs/usage/nfd-topology-updater.md

```diff
@@ -20,5 +20,9 @@ When run as a daemonset, nodes are re-examined for the allocated resources
 (to determine the information of the allocatable resources on a per zone basis
 where a zone can be a NUMA node) at an interval specified using the
 [`-sleep-interval`](../reference/topology-updater-commandline-reference.html#-sleep-interval)
-option. The default sleep interval is set to 60s which is the the value when no
+option. The default sleep interval is set to 60s which is the value when no
 -sleep-interval is specified.
+In addition, it can avoid examined specific allocated resources
+given a configuration of resources to exclude via [`-config`](../reference/topology-updater-commandline-reference.html#-config)
```

Comment:
Should we point directly to `../reference/topology-updater-configuration-reference.html#excludelist` instead?

We probably want to have a separate paragraph/sub-section about configuration, though. State that nfd-topology-updater has a config file and NOTE that unlike nfd-worker, dynamic updates are not supported ATM